deepedit/deepedit_bundle/configs/train.json

```diff
@@ -0,0 +1,355 @@
+                },
+                {
+                    "_target_": "AsDiscreted",
+                    "keys": ["pred", "label"],
+                    "argmax": [true, false],
+                    "to_onehot": "$len(@label_names)+1"
+                }
+            ]
+        },
+        "handlers": [
+            {
+                "_target_": "LrScheduleHandler",
+                "lr_scheduler": "@lr_scheduler",
+                "print_lr": true
+            },
+            {
+                "_target_": "ValidationHandler",
+                "validator": "@validate#evaluator",
+                "epoch_level": true,
+                "interval": 1
+            },
+            {
+                "_target_": "StatsHandler",
+                "tag_name": "train_loss",
+                "output_transform": "$monai.handlers.from_engine(['loss'], first=True)"
+            },
+            {
+                "_target_": "TensorBoardStatsHandler",
+                "log_dir": "@output_dir",
+                "tag_name": "train_loss",
+                "output_transform": "$monai.handlers.from_engine(['loss'], first=True)"
+            }
+        ],
+        "key_metric": {
+            "train_dice": {
+                "_target_": "MeanDice",
+                "output_transform": "$monai.handlers.from_engine(['pred', 'label'])"
+            }
+        },
+        "train_iteration_update": {
+             "_target_": "Interaction",
+            "deepgrow_probability": "@deepgrow_probability_train",
+            "transforms": "@train#click_transforms",
+            "click_probability_key": "probability",
+            "train": true,
+            "label_names": "@label_names"
+
+        },
+        "trainer": {
+            "_target_": "SupervisedTrainer",
+            "device": "@device",
+            "max_epochs": 500,
+            "train_data_loader": "@train#dataloader",
+            "network": "@network",
+            "optimizer": "@optimizer",
+            "loss_function": "@loss",
+            "inferer": "@train#inferer",
+            "amp": true,
+            "postprocessing": "@train#post_transforms",
+            "key_train_metric": "@train#key_metric",
+            "train_handlers": "@train#handlers",
+            "iteration_update": "@train#train_iteration_update"
+        }
+    },
+    "validate": {
+        "pre_transforms": {
+            "_target_": "Compose",
+            "transforms": [
+                "%train#pre_transforms#transforms#0",
+                "%train#pre_transforms#transforms#1",
+                "%train#pre_transforms#transforms#2",
+                "%train#pre_transforms#transforms#3",
+                "%train#pre_transforms#transforms#4",
+                "%train#pre_transforms#transforms#10",
+                "%train#pre_transforms#transforms#11",
+                "%train#pre_transforms#transforms#12",
+                "%train#pre_transforms#transforms#13",
+                "%train#pre_transforms#transforms#14"
```

Comment:
any idea how to simplify this? perhaps put 0-4 and 10-14 to separate .json files? cc@Nic-Ma the current one is not easy to read...

Reply:
One way to do this is have a separate definition called `common_transforms` which is just a list of transforms common to both training and validation, then textually expand that list here using the % sigil and in `@train#pre_transforms`.

Reply:
I will update the spleen bundle example for this part.

Thanks.

Reply:
Hi @ericspod ,

Submitted a PR for it: https://github.com/Project-MONAI/tutorials/pull/710.
Could you please help review it?

Thanks in advance.

Reply:
Hi @diazandr3s ,

Could you please help update this bundle example referring to the updated spleen bundle?

Thanks in advance.

Reply:
Hi @diazandr3s ,

For your case, I think you can chain the transforms for the trainer like:
```json
"preprocessing": {
    "_target_": "Compose",
    "transforms": "$@train#deterministic_trans[0: 5] + @train#random_trans + deterministic_trans[5: 10]"
}
```
And BTW, I already added you to the `model-zoo` repo, could you please help submit this bundle to that bundle?
https://github.com/Project-MONAI/model-zoo
Then let's close this PR and discuss in `model-zoo` directly.

Thanks in advance.

Reply:
Thanks, @Nic-Ma
Will modify this and create the PR there :)